am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/ResourceMetadata.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.amlib.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
```

Comment:
Please try hiding constructor. I would like to hide constructors to enforce use of builders. We start here and apply everywhere later.